pyproject.toml

```diff
@@ -5,6 +5,7 @@ build-backend = "maturin"
 [project]
 name = "y-py"
 description = "Python bindings for the Y-CRDT built from yrs (Rust)"
+dynamic = ["version"]
```

Comment:
Added this to fix `maturin develop` error:
```
💥 maturin failed
  Caused by: `project.version` field is required in pyproject.toml unless it is present in the `project.dynamic` list
```

Did not want to duplicate version already defined in `Cargo.toml`.